README.md

```diff
@@ -1,2 +1,32 @@
-# flexvolume-nodejs
-Azure flex-volume to nodejs property integration.
+# properties-volume-nodejs
+[![MIT license](http://img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
+<br>[![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/properties-volume-nodejs.svg)](https://greenkeeper.io/)
```

Comment:
remove please, we use dependabot now instead

Reply:
ok np.